pkg/hardwareutils/bmc/redfish.go

```diff
@@ -8,7 +8,7 @@ import (
 
 func init() {
 	schemes := []string{"http", "https"}
-	RegisterFactory("redfish", newRedfishAccessDetails, schemes)
+	RegisterFactory(redfish, newRedfishAccessDetails, schemes)
 	RegisterFactory("ilo5-redfish", newRedfishAccessDetails, schemes)
 	RegisterFactory("idrac-redfish", newRedfishiDracAccessDetails, schemes)
```

Comment:
Same question, it's weird to have one thing as a constant and the two other as plain strings.

Reply:
I'm going to guess it is because the linter complains only after you have N occurrences of the same string.

Reply:
Yep, understood. Still, I'd prefer us to use constants consistently, regardless of what the linter forces us to.

Reply:
True, I will send a patch